### ISSUE TITLE
Support atom_author

### ISSUE THREAD

**Issue by simonw** [simonw/datasette-atom#9]
https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Ftil.simonwillison.net%2Ftil%2Ffeed.atom%3F_feed_title%3DSimon%2520Willison%2520TIL complains about missing author elements.

**Comment by simonw** [comment]
https://validator.w3.org/feed/docs/atom.html#person:

> `<author>` and `<contributor>` describe a person, corporation, or similar entity. It has one required element, `name`, and two optional elements: `uri`, `email`.
> 
> `<name>` conveys a human-readable name for the person.
> 
> `<uri>` contains a home page for the person.
> 
> `<email>` contains an email address for the person.

So I'll add code to spot `author_name`, `author_uri` and `author_email` columns.

**Comment by simonw** [comment]
Released in 0.5. Documentation here: https://github.com/simonw/datasette-atom/blob/1e2578efd55849261938a55e794e78c15cccd8cd/README.md#usage

**Comment by simonw** [comment]
https://til.simonwillison.net/til/feed.atom?_feed_title=Simon%20Willison%20TIL demonstrates this in action:

<img width="634" alt="Mozilla_Firefox" src="https://user-images.githubusercontent.com/9599/80734230-b8171e00-8ac3-11ea-80c3-423b018531d5.png">

Defined using this SQL query: https://til.simonwillison.net/til/feed
```sql
SELECT
  'tag:til.simonwillison.net,2020-04-30:' || path as atom_id,
  title as atom_title,
  url as atom_link,
  created_utc as atom_updated,
  render_markdown(body) as atom_content_html,
  'Simon Willison' as atom_author_name,
  'https://simonwillison.net/' as atom_author_uri
FROM
  til
order by
  created_utc desc
limit
  15
```
